pkg/security/security_profile/profile/manager.go

```diff
@@ -588,45 +596,41 @@ func (m *SecurityProfileManager) LookupEventInProfiles(event *model.Event) {
 }
 
 // tryAutolearn tries to autolearn the input event. Returns true if the event was autolearned.
```

Comment:
Should we add a tiny comment to explain the 3 output variables ? (at least the first 2)